modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -1078,6 +1099,12 @@ definition {
 				value1 = ${secretAction});
 		}
 
+		if (IsElementPresent(locator1 = "ObjectAction#ADD_ERROR_MESSAGE_VALIDATION")) {
```

Comment:
Can this be changed to isSet(errorMessage) to keep pattern of the macro?

Reply:
On second look, I think this might already be on master. If that's the case, we can keep as is.